src/main/java/org/broadinstitute/hellbender/utils/bwa/BwaMemPairEndOrientation.java

```diff
@@ -0,0 +1,8 @@
+package org.broadinstitute.hellbender.utils.bwa;
+
+/**
+ * Pair-end read orientations where "F" stands for forward and "R" for reverse.
+ */
+public enum BwaMemPairEndOrientation {
+    FF, FR, RF, RR
```

Comment:
Please add some comments to these to make their meaning clearer.  Something like:
"righties" -- both reads of the pair align to + reference strand
"innies" -- pair aligns with upstream read on + strand, downstream read on - strand
etc.